apps/publikator/components/Auth/SignIn.js

```diff
@@ -238,11 +239,11 @@ class SignIn extends Component {
           </div>
         </form>
         <Label {...styles.hint}>
-          <Link route='legal/privacy'>
+          <Link href='/legal/privacy' passHref>
             <a {...styles.hintA}>{t('signIn/privacy')}</a>
           </Link>
```

Comment:
this is a outdated route that never worked in publikator anyhow. let's fix it when changing (maybe also apply in admin tool)

suggestion just as potential solution, needs const import to work:

```suggestion
            <a {...styles.hintA} href={`${FRONTEND_BASE_URL}/datenschutz`}>{t('signIn/privacy')}</a>
```